src/components/Slider.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, createElement, DOM } from "react";
+
+import * as classNames from "classnames";
+import * as RcSlider from "rc-slider";
+
+import "rc-slider/dist/rc-slider.css";
+import "../ui/Slider.css";
+
+import { Alert } from "./Alert";
+
+export interface SliderProps {
```

Comment:
It's a big pile of properties. Maybe grouping them make sense? Some are settings (like `maxvalue`, `minValue`) while others provide callbacks when something happens (`onChange`, `onUpdate`).